cmd/query/app/server.go

```diff
@@ -74,12 +81,18 @@ func NewServer(querySvc *querysvc.QueryService,
 		return nil, errors.New("server with TLS enabled can not use same host ports for gRPC and HTTP.  Use dedicated HTTP and gRPC host ports instead")
 	}
 
-	grpcServer, err := createGRPCServer(querySvc, metricsQuerySvc, options, tm, telset)
+	var grpcServer *grpc.Server
+	if separatePorts {
```

Comment:
we need to add warning log (when parsing CLI flags) that shared port will not be supported going forward

Reply:
done